analyses/molecular-subtyping-embryonal/02-generate-subset-files.R

```diff
@@ -3,8 +3,8 @@
 # This script subsets the files required for subtyping non-MB and non-ATRT
 # embryonal tumors. A sample will be included if either of the following
 # conditions are met: 1) the sample is labeled as an embryonal tumor
-# (broad_histology) but NOT an MB or ATRT tumor (pathology_diagnosis) OR 2)
-# the sample contains a TTYH1 fusion (5' partner)
+# but NOT an MB or ATRT tumor (pathology_diagnosis) OR 2) the sample contains
+# a TTYH1 fusion (5' partner)
```

Comment:
I think if you are not going to point a reader to the `01` notebook here and leave it at that, this needs to be more detailed - it's a lot more complicated than this now!